Source/WebKit/UIProcess/ios/WebDataListSuggestionsDropdownIOS.mm

```diff
@@ -464,7 +464,9 @@ - (void)_displayWithActivationType:(WebCore::DataListSuggestionActivationType)ac
 
     [self _updateTextSuggestions];
 
-    if (![UIKeyboard isInHardwareKeyboardMode] && !(activationType == WebCore::DataListSuggestionActivationType::IndicatorClicked || activationType == WebCore::DataListSuggestionActivationType::DataListMayHaveChanged))
+    if (!_suggestionsContextMenuPresenter
```

Comment:
it might be a bit nicer as a lambda that returns true/false instead of a giant if statement?

Reply:
I'll do this, since it looks like I'll also need to add a `USE(UICONTEXTMENU)`.